libavcodec/cbs_h2645.c

```diff
@@ -2165,6 +2165,24 @@ static const SEIMessageTypeDescriptor cbs_sei_common_types[] = {
         sizeof(SEIRawAmbientViewingEnvironment),
         SEI_MESSAGE_RW(sei, ambient_viewing_environment),
     },
+    {
+        SEI_TYPE_FILM_GRAIN_CHARACTERISTICS,
+        1, 0,
+        sizeof(SEIRawFilmGrainCharacteristics),
+        SEI_MESSAGE_RW(sei, film_grain_characteristics),
+    },
+    {
+        SEI_TYPE_DISPLAY_ORIENTATION,
```

Comment:
It's not included in h274, right?

Reply:
Nope, it is. See 8.25 Display orientation SEI message

Reply:
8.25? not in vvc spec, right

Reply:
It's in the 
ITU-T H.274 (V3)
- 8.25 Display orientation SEI message

Reply:
ok, can we rename H2645SEIDisplayOrientation and use it?